TheEthicalHackerCup/Assets/Tutorial/Scripts/Tutorial.cs

```diff
@@ -5,57 +5,66 @@
 
 public class Tutorial : MonoBehaviour
 {
-    [SerializeField] private Boolean tutorialActive;
-    [SerializeField] private List<GameObject> tutorialStates;
     private Canvas tutorialCanvas;
 
-    private int tutorialStateIndex;
+    [SerializeField] private GameObject tutorialSpeech;
+    [SerializeField] private GameObject tutorialPointer;
+    [SerializeField] private TextAsset pointerConfig;
+    private string[] pointerLines;
+    private int pointerIndex;
+    private string pointer;
+    private GameObject tutorialPointerCopy;
 
     // Start is called before the first frame update
     void Start()
     {
         tutorialCanvas = this.gameObject.GetComponent<Canvas>();
-        tutorialStateIndex = 0;
-        hideTutorialScenes();
+        tutorialSpeech = Instantiate(tutorialSpeech, transform);
+        tutorialSpeech.transform.localPosition = new Vector3(-10, -80, 0);
+
+        pointerLines = pointerConfig.text.Split('\n');
+        pointerIndex = 0;
+        
+        pointer = pointerLines[pointerIndex];
+        if (pointer.Trim() != "") {
+            tutorialPointerCopy = Instantiate(tutorialPointer, transform);
+            tutorialPointerCopy.transform.localPosition = getPointerPosition(pointer);
+            tutorialPointerCopy.transform.localRotation = getPointerRotation(pointer);
+        }
+        pointerIndex += 1;
```

Comment:
This code block is repeated in start() and in update()

Factor it out into a private method